parm/config/config.fcst

```diff
@@ -252,9 +252,8 @@ export DO_OCN_SPPT=${DO_OCN_SPPT:-"NO"}
 export DO_OCN_PERT_EPBL=${DO_OCN_PERT_EPBL:-"NO"}
 
 #coupling settings
-export FRAC_GRID=".true."
 export cplmode="nems_frac"
-if [ $FRAC_GRID = ".false." ]; then
+if [[ "${FRAC_GRID:-".true."}" = ".false." ]]; then
```

Comment:
Does this work as intended? Seems like there might be a quotation mark issue on the left side.